mocks/service/github/index.js

```diff
@@ -14,6 +16,21 @@ responses['okay-new-hook-created'] = (req, res) => {
   res.status(201);
   // No response body implemented
   res.send();
+
+  // Send ping to webhook endpoint
+  request.post({
+    url: req.body.config.url,
+    headers: {
+      'X-Github-Delivery': '72d3162e-cc78-11e3-81ab-4c9367dc0958',
+      'User-Agent': 'GitHub-Hookshot/044aadd',
+      'X-GitHub-Event': 'issues'
```

Comment:
[push](https://developer.github.com/v3/activity/events/types/#pushevent) would be a more realistic choice of event.